src/dc_receive_imf.rs

```diff
@@ -102,28 +100,14 @@ pub fn dc_receive_imf(
         sent_timestamp = mailparse::dateparse(value).unwrap_or_default();
     }
 
-    // Make sure, to_ids starts with the first To:-address (Cc: is added in the loop below pass)
-    if let Some(field) = mime_parser.get(HeaderDef::To) {
-        dc_add_or_lookup_contacts_by_address_list(
-            context,
-            &field,
-            if !incoming {
-                Origin::OutgoingTo
-            } else if incoming_origin.is_verified() {
-                Origin::IncomingTo
-            } else {
-                Origin::IncomingUnknownTo
-            },
-            &mut to_ids,
-        )?;
-    }
-
```

Comment:
it was wrong to first look at "To" isntead of From because the latter modifies the incoming_origin.  Therefore, added contacts remained below the threshold where they would be shown when you want to create a chat or add group members.